src/main/java/com/faforever/client/preferences/ChatPrefs.java

```diff
@@ -35,6 +38,7 @@
 
 
   public ChatPrefs() {
+    ukTime = new SimpleStringProperty(TimeInfo.AUTO.getDisplayName());
```

Comment:
Preferences should not store the display name, but the technical names (in this cause `AUTO.name()`).